src/bin/pgcopydb/defaults.h

```diff
@@ -81,6 +82,7 @@
 #define POSTGRES_PING_RETRY_BASE_SLEEP_TIME 5         /* milliseconds */
 
 #define POSTGRES_PORT 5432
+#define POSTGRES_BLOCK_SIZE 8192
```

Comment:
Searching for the Postgres source code definition of that value, it seems that it is dynamic enough (compile-time?) to warrant actually asking the database for the value. It now seems best to have that computation done in the SQL query we run on the source database, adding a new column with the result of `relpages * current_setting('block_size')::bigint` or something.

Reply:
This value is set at PG configure. Allowed values are (in kB): 1,2,4,8,16,32.

```
./configure --with-blocksize=BLOCKSIZE
```

In the worst case, we end up overestimating by a factor of 8, or underestimate by a factor of 4.

Let me update the SQL commands to fetch the current value.